plugin/trino-atop/pom.xml

```diff
@@ -153,4 +153,31 @@
             <scope>test</scope>
         </dependency>
     </dependencies>
+
+    <build>
+        <plugins>
+            <plugin>
+                <groupId>org.apache.maven.plugins</groupId>
+                <artifactId>maven-surefire-plugin</artifactId>
+                <dependencies>
```

Comment:
Why not to add this to the main pom.xml?

Reply:
per @losipiuk this would blow up modules which do not have testng on the classpath.

Reply:
It is better to blow up loudly, than not run tests silently IMHO

Reply:
I agree with you and can do the change in the top level pom if only we have agreement to have testng always on the test classpath until it is no longer used anywhere.

Reply:
I already tried that. Surefire fails if there are no tests to run in a module.

Reply:
I think it's more complicated than that.
When i put 
```
<build>
        <plugins>
            <plugin>
                <groupId>org.apache.maven.plugins</groupId>
                <artifactId>maven-surefire-plugin</artifactId>
                <dependencies>
                    <!-- allow both JUnit and TestNG -->
                    <dependency>
                        <groupId>org.apache.maven.surefire</groupId>
                        <artifactId>surefire-junit-platform</artifactId>
                        <version>${dep.plugin.surefire.version}</version>
                    </dependency>
                    <dependency>
                        <groupId>org.apache.maven.surefire</groupId>
                        <artifactId>surefire-testng</artifactId>
                        <version>${dep.plugin.surefire.version}</version>
                    </dependency>
                    <dependency>
                        <groupId>org.junit.jupiter</groupId>
                        <artifactId>junit-jupiter-engine</artifactId>
                        <version>${dep.junit.version}</version>
                    </dependency>
                </dependencies>
            </plugin>
        </plugins>
    </build>
```

into `trino-teradata-functions` module which has only Junit tests, surefire run 14 junit tests, then it run 0 testng tests and then it did not complain that no tests were found.